docs/zilevm/onboard/onboard-metamask.md

```diff
@@ -25,55 +25,59 @@ description: Wallets that can interact with ZILEVM
 
 !!! tip
 
-    You can send funds between ZILEVM and Zilliqa networks by translating
-    your Zilliqa address into it's base 16 representation which ZILEVM and
+    You can send funds between EVM and Zilliqa format APIs translating
+    your Zilliqa address into it's base 16 representation which
     Metamask can understand.
 
 ### Configure MetaMask with ZILEVM network
 
 To add a new network to MetaMask - click the current network selected at the top
 of the extenstion and press 'Add Network'.
 
-Enter the below configuration, and press save.
+Enter the below configuration:
 
-| Network Type | Network Name | Network RPC                                                        | ChainID | Currency Symbol | Block Explorer URL                                   |
-| ------------ | ------------ | ------------------------------------------------------------------ | ------- | --------------- | ---------------------------------------------------- |
-| Testnet      | EVM Test     | [https://evm-api-dev.zilliqa.com](https://evm-api-dev.zilliqa.com) | 33101   | ZIL             | [https://evmx.zilliqa.com](https://evmx.zilliqa.com) |
-| Mainnet      |              |                                                                    |         |                 |                                                      |
+| Network Type | Network Name | Network RPC                                                | ChainID | Currency Symbol | Block Explorer URL                                           |
+| ------------ | ------------ | ---------------------------------------------------------- | ------- | --------------- | ------------------------------------------------------------ |
+| Testnet      | EVM Test     | [https://api-dev.zilliqa.com](https://api-dev.zilliqa.com) | 33101   | ZIL             | [https://evmx-dev.zilliqa.com](https://evmx-dev.zilliqa.com) |
+| Mainnet      | EVM Mainnet  | [https://api.zilliqa.com](https://api.zilliqa.com)         | 32769   | ZIL             | [https://evmx.zilliqa.com](https://evmx.zilliqa.com)]        |
 
 ### Sending funds
 
-Do not send Zilliqa NFTs or fungible tokens to ZILEVM addresses and vice-versa. You will lose your assets.
+Do not send Zilliqa NFTs or fungible tokens to EVM addresses and vice-versa. You will lose your assets - see the [FAQ](../../faq/faq-introduction).
 
 #### Converting addresses
 
-Navigate to [https://devex.zilliqa.com/address/](https://devex.zilliqa.com/address/)address}` either using a ZIL bech32 address or an ZILEVM base16 address.
+Navigate to [https://devex.zilliqa.com/address/](https://devex.zilliqa.com/address/)address}` either using a ZIL bech32 address or a base16 address.
```

Comment:
Something looks off in this sentence - I am aware that this technically just inherited debt, not your PR, but maybe something like this is more accurate:
```suggestion
Navigate to [https://devex.zilliqa.com/address/](https://devex.zilliqa.com/address/) to convert using either using a ZIL bech32 address or a base16 address.
```